apps/gimbtonomy/LEDMatrix.cpp

```diff
@@ -0,0 +1,47 @@
+/* LED Matrix Source File */
+
+#include "LEDMatrix.h"
+
+
+//Data constructor
+//LEDMatrix :: LEDMatrix (void)
+//{
+    //initialize PWM pins - pins tbd
+    
+    //redPWM(/*redLED*/);
+    //greenPWM(/*greenLED*/);
+    //bluePWM(/*blueLED*/);
+//}
+
+//Set how often it flashes/sec
+void LEDMatrix :: setFrequency(uint8_t msgFrequency)
+{
+    redPWM.period(1/msgFrequency);
+    greenPWM.period(1/msgFrequency);
+    bluePWM.period(1/msgFrequency);
```

Comment:
the periods need to be std::chrono::duration. See https://en.cppreference.com/w/cpp/chrono/duration